src/templates/week.tsx

```diff
@@ -41,7 +44,7 @@ type Props = {
 }
 
 type Lesson = ContentfulLesson & {
-  bookmarked?: boolean
+  bookmarked?: any
```

Comment:
why change bookmarked from boolean to any? This is displayed as a warning, and the bookmarked value should never be anything else other than boolean

Reply:
Oh! I was playing around with things and I forgot to replace it. Just fixed it!